docker-vertica/packages/cleanup.sh

```diff
@@ -23,6 +23,7 @@
 #
 # wander around in the image looking for things you can remove
 rm -r -f \
+   /opt/vertica/config/* \
```

Comment:
This might be problematic for vertica-demo. They use the vertica-k8s image, and probably rely on the config directory to be set up. Can we change this to remove specific things in `/config`? The agent keys, the keys for the https_certs, etc.

Reply:
Ok I will delete specific keys files.